modules/caddyhttp/reverseproxy/fastcgi/client_test.go

```diff
@@ -135,16 +134,19 @@ func sendFcgi(reqType int, fcgiParams map[string]string, data []byte, posts map[
 			length = len(data)
 			rd := bytes.NewReader(data)
 			resp, err = fcgi.Post(fcgiParams, "", "", rd, int64(rd.Len()))
+			resp.Body.Close()
```

Comment:
You might want to wait until the `Body` is read below on line 170 before closing it, no ?